### ISSUE TITLE
PushState

### ISSUE THREAD

**Issue by kersten** [Saquib764/dna-router#3]
Hi,

could you implement the HTML5 PushState, so we don't need the hash

Greetings

**Comment by Saquib764** [comment]
Yes. I am already implementing HTML5 Pushstate for the stable version.
I'll be publishing it within next 20 days or so.

**Comment by kersten** [comment]
very nice, thanks

**Comment by Saquib764** [comment]
:)
